webapp-django/crashstats/settings/bundles.py

```diff
@@ -0,0 +1,468 @@
+    'manage:supersearch_fields': {
+        'source_filenames': (
+            'manage/js/supersearch_fields.js',
+        ),
+        'output_filename': 'js/manage-supersearch-fields.min.js',
+    },
+    'manage:symbols_uploads': {
+        'source_filenames': (
+            'manage/js/symbols-uploads.js',
+        ),
+        'output_filename': 'js/manage-symbols-uploads.min.js',
+    },
+    'manage:users': {
+        'source_filenames': (
+            'manage/js/users.js',
+        ),
+        'output_filename': 'js/manage-users.min.js',
+    },
+    'profile': {
+        'source_filenames': (
+            'profile/js/profile.js',
+        ),
+        'output_filename': 'js/profile.min.js',
+    },
+    'signature_report': {
+        'source_filenames': (
+            'signature/js/signature_report.js',
+            'signature/js/signature_tab.js',
+            'signature/js/signature_tab_summary.js',
+            'signature/js/signature_tab_graphs.js',
+            'signature/js/signature_tab_reports.js',
+            'signature/js/signature_tab_aggregations.js',
+            'signature/js/signature_tab_comments.js',
+            'signature/js/signature_tab_graph.js',
+            'signature/js/signature_panel.js',
+        ),
+        'output_filename': 'js/signature-report.min.js',
+    },
+    'search_custom': {
+        'source_filenames': (
+            'supersearch/js/lib/ace/ace.js',
+            'supersearch/js/lib/ace/theme-monokai.js',
+            'supersearch/js/lib/ace/mode-json.js',
+            'supersearch/js/socorro/search_custom.js',
+        ),
+        'output_filename': 'js/search-custom.min.js',
+    },
+    'search': {
+        'source_filenames': (
+            'supersearch/js/socorro/search.js',
+        ),
+        'output_filename': 'js/search.min.js',
+    },
+    'tokens': {
+        'source_filenames': (
+            'tokens/js/home.js',
+        ),
+        'output_filename': 'js/tokens.min.js',
+    },
+}
+
+# This is sanity checks, primarily for developers. It checks that
+# you haven't haven't accidentally make a string a tuple with an
+# excess comma, no underscores in the bundle name and that the
+# bundle file extension is either .js or .css.
+# We also check, but only warn, if a file is re-used in a different bundle.
+# That's because you might want to consider not including that file in the
+# bundle and instead break it out so it can be re-used on its own.
+_used = {}
+for config in PIPELINE_JS, PIPELINE_CSS:  # NOQA
+    _trouble = set()
+    for k, v in config.items():
+        assert isinstance(k, basestring), k
+        out = v['output_filename']
+        assert isinstance(v['source_filenames'], tuple), v
+        assert isinstance(out, basestring), v
+        assert '_' not in out
+        assert out.endswith('.min.css') or out.endswith('.min.js')
+        for asset_file in v['source_filenames']:
+            if asset_file in _used:
+                # Consider using warnings.warn here instead
```

Comment:
The whole point of this stuff is to help developers hacking on the code. E.g. you and me. Thus isn't never ever going to be run in production or in run-time. So the ideal place for it is `stdout`. No doubt. 

However, normally in these kinds of cases you use `warnings.warn()` which, by default, yields to `stdout`. 
But, the benefit of doing...:
```python
import warnings
warnings.warn('message')
```
is that that message will only **once** be printed on `stdout`. However, that's only true once the python process has started and kept running. But when you do things like `./manage.py runserver` and change a file it will **restart** the python process each time. The same is true if you run the test repeatedly. It restarts the python process. So we're not getting the benefit of warnings not being repeated. 

Lastly, using `warnings.warn` (or `logger.info` for that matter) is much more verbose. With `warnings.warn('message')` you get...:
```
/Users/peterbe/dev/MOZILLA/SOCORRO/socorro/webapp-django/crashstats/settings/bundles.py:459: UserWarning: message
  warnings.warn('message')
```
That's a lot noisier than just doing `print 'message'`. 

One thing we should consider is perhaps not bother with ANY of these if you're NOT in debug mode.